src/main/java/io/odpf/depot/message/proto/ProtoOdpfParsedMessage.java

```diff
@@ -24,6 +31,12 @@
 public class ProtoOdpfParsedMessage implements ParsedOdpfMessage {
     private final DynamicMessage dynamicMessage;
 
+    private JsonFormat.Printer jsonPrinter = JsonFormat.printer()
+            .omittingInsignificantWhitespace()
+            .preservingProtoFieldNames()
```

Comment:
we should add explicit tests for all these three flags

Reply:
have added tests for preserving protofields option.